### ISSUE TITLE
Dropdown: Screen reader is not reading the “Role and State” information  for dropdown.

### ISSUE THREAD

**Issue by sagaranandani17** [OfficeDev/office-ui-fabric-react#11092]
### Environment Information

- **Package version(s)**: 7.24.0
- **OS version:** 1903 (OS Build: 18326.295)
- **Browser:** Edge(Microsoft Edge 44.18362.267.0)

**User Experience:** 
If the screen reader is not reading the “Role and State” for dropdown than screen reader users will not be able to find out the role and state  property of controls and will face difficulties while interacting with it.

**Actual behavior:**
 Screen reader is not reading the “Role & State” properties for dropdown .

**Expected behavior:**
 Screen reader should read the “Role & State” properties for dropdown, screen reader should read the role(listbox) and state when focus is on dropdown using tab. Please find the attached screenshot.

**Please find the attachment**
![image](https://user-images.githubusercontent.com/57341535/68287471-ca7e4500-00a8-11ea-831c-1936e8d40ebf.png)

**Comment by JasonGore** [comment]
The behavior of "Basic uncontrolled example" is the same as [this ARIA collapsible listbox example](https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html).

The following information is read when focused:
> Basic uncontrolled example Apple, Apple selected, selection contains <say-as interpret-as="spell">7</say-as> items, 

Which is similar to the ARIA collapsible listbox example:
> Choose an element: Neptunium, button, 

In both cases, role and state of `listbox` is not announced since the `listbox` is not expanded.

Then when expanding the listbox, Dropdown announces the `list` role:
> Basic uncontrolled example list, Apple, <say-as interpret-as="spell">1</say-as> of <say-as interpret-as="spell">7</say-as>, selected, 

Similar to the ARIA collapsible listbox example, which also narrates role:
> Choose an element: list, Neptunium, <say-as interpret-as="spell">1</say-as> of 26, selected, 

It seems to me that both implementations are symmetrical. If there is an issue here, it appears to be with Narrator and not with Fabric.

**Comment by JasonGore** [comment]
I don't think this will autoclose as I intended. I will close manually. As it's been a week since I last posted, please feel free to let me know if you have any additional concerns. Thanks!
